src/settings.ini

```diff
@@ -58,4 +58,4 @@ mod4LAsTab=0
 # show debug output in a separate console window
 # Debug-Ausgabe in einem separaten Konsolen-Fenster anzeigen
 # (wenn neo-llkh.exe in der Git Bash gestartet wird, sollte dieser Wert auf 0 gesetzt werden)
-debugWindow=0
+debugWindow=1
```

Comment:
`capsLockEnabled` standardmäßig zu aktivieren finde ich gut. `debugWindow` will ich aber deaktiviert lassen, weil man das nur zur Fehleranalyse braucht. (Ich kann das aber auch mit einem nachträglichen Commit wieder zurück ändern.)

Reply:
ja, eigentlich wollte ich die settings nicht wirklich in den pull request nehmen. debugWindow macht Sinn, standardmässig deaktiviert zu lassen